README.rst

```diff
@@ -200,15 +200,15 @@ intend to install multiple versions using the same prefix you must decide which
 version (if any) is your "primary" version.  Install that version using
 ``make install``.  Install all other versions using ``make altinstall``.
 
-For example, if you want to install Python 2.7, 3.6, and 3.13 with 3.13 being the
-primary version, you would execute ``make install`` in your 3.13 build directory
+For example, if you want to install Python 2.7, 3.6, and 3.14 with 3.14 being the
+primary version, you would execute ``make install`` in your 3.14 build directory
 and ``make altinstall`` in the others.
 
 
 Release Schedule
 ----------------
 
-See :pep:`719` for Python 3.13 release details.
+See :pep:`745` for Python 3.14 release details.
```

Comment:
This shows up like this on GitHub:

> See `745` for Python 3.14 release details.

https://github.com/Wulian233/cpython/blob/readme/README.rst#release-schedule

And I don't think this file is processed by Sphinx for the main docs?

Let's create a manual link to the PEP instead:

```suggestion
See `PEP 745 <https://peps.python.org/pep-0745/>`__ for the Python 3.14 release details.
```